### PR TITLE
staticaddr: `loop static listswaps`

### PR DESCRIPTION
**Pull request by hieblmi**
_builds on https://github.com/lightninglabs/loop/pull/786_

_only the last commit is relevant for this PR_

This PR introduces `loop static listswaps` to enable info look-up of static address loop-in swaps.

Available look-up commands:
```
loop static listsunspent (looks up lnd wallet utxos for the static address)
loop static summary
loop static listdeposits
loop static listswaps
```

### CODE REVIEW

File: loopd/swapclient_server.go [hunk 1]
```diff
@@ -1444,15 +1444,13 @@ func (s *swapClientServer) WithdrawDeposits(ctx context.Context,
 	}, err
 }
 
-// GetStaticAddressSummary returns a summary static address related information.
-// Amongst deposits and withdrawals and their total values it also includes a
-// list of detailed deposit information filtered by their state.
-func (s *swapClientServer) GetStaticAddressSummary(ctx context.Context,
-	req *looprpc.StaticAddressSummaryRequest) (
-	*looprpc.StaticAddressSummaryResponse, error) {
+func (s *swapClientServer) ListStaticAddressDeposits(ctx context.Context,
```
Comment (bhandras) [inline loopd/swapclient_server.go:1447]: nit: missing godoc

File: loopd/swapclient_server.go [hunk 2]
```diff
@@ -1463,69 +1461,116 @@ func (s *swapClientServer) GetStaticAddressSummary(ctx context.Context,
...
+				int64(params.Expiry) - bestBlockHeight
 	}
 
-	loopIn, err := s.staticLoopInManager.DeliverLoopInRequest(ctx, req)
+	return &looprpc.ListStaticAddressDepositsResponse{
+		FilteredDeposits: filteredDeposits,
+	}, nil
+}
+
+func (s *swapClientServer) ListStaticAddressSwaps(ctx context.Context,
```
Comment (bhandras) [inline loopd/swapclient_server.go:1516]: nit: missing godoc

**Review by bhandras — COMMENTED** [review]
New commit LGTM 🎉

**Review by bhandras — APPROVED** [review]

File: loopd/swapclient_server.go (hunk 1 above)
Comment (hieblmi) [reply loopd/swapclient_server.go:1450]: fixed

**Pull request merged**